### ISSUE TITLE
Use this project on Windows by pairing to Mac

### ISSUE THREAD

**Issue by maonaoda** [Redth/DotNet.Platform.SlimBindings#4]
Can i use this project on Windows by pairing to Mac ?

> Severity	Code	Description	Project	File	Line	Suppression State
> Error	MSB3073	The command "xcodebuild -project E:\Projects\SlimBindings\firebase\macios\native\analytics\MauiFirebaseAnalytics.xcodeproj archive -scheme MauiFirebaseAnalytics -configuration Release -archivePath E:\Projects\SlimBindings\firebase\macios\native\analytics\.build\MauiFirebaseAnalytics-ios.xcarchive -destination 'generic/platform=iOS'  -derivedDataPath E:\Projects\SlimBindings\firebase\macios\native\analytics\.build -IDECustomBuildProductsPath='' -IDECustomBuildIntermediatesPath='' ENABLE_BITCODE=NO SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES OTHER_SWIFT_FLAGS='-no-verify-emitted-module-interface' OBJC_CFLAGS='-fno-objc-msgsend-selector-stubs -ObjC'" exited with code 9009.	FirebaseAnalytics.Binding	E:\Projects\SlimBindings\eng\Common.macios.targets	40	
> Error	MSB3073	The command "xcodebuild -project E:\Projects\SlimBindings\firebase\macios\native\messaging\MauiFirebaseMessaging.xcodeproj archive -scheme MauiFirebaseMessaging -configuration Release -archivePath E:\Projects\SlimBindings\firebase\macios\native\messaging\.build\MauiFirebaseMessaging-ios.xcarchive -destination 'generic/platform=iOS'  -derivedDataPath E:\Projects\SlimBindings\firebase\macios\native\messaging\.build -IDECustomBuildProductsPath='' -IDECustomBuildIntermediatesPath='' ENABLE_BITCODE=NO SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES OTHER_SWIFT_FLAGS='-no-verify-emitted-module-interface' OBJC_CFLAGS='-fno-objc-msgsend-selector-stubs -ObjC'" exited with code 9009.	FirebaseMessaging.Binding	E:\Projects\SlimBindings\eng\Common.macios.targets	40	
>

**Comment by gabsamples6** [comment]
Same issue - Cannot run on windows - we use windows and we pair to mac

**Comment by aritchie** [comment]
This was always going to be the case.  The native frameworks can only be built on a mac because of Apple requirements, not Microsoft.

**Comment by Redth** [comment]
It _might_ be possible to make this eventually work with pair to mac, but for now it will not.  The custom targets used to create the xcframeworks for the slim bindings are not written to use any of the remote pair to mac abilities.  It is likely possible to make the xcode build invocations use tasks that can run remotely, but there'd need to be more work to make the xcode project files copied over as part of this process since they need to build there.

tl;dr - this isn't currently in scope
